src/components/GeneralPage.tsx

```diff
@@ -31,8 +31,10 @@ import { dismissPrivacyAlert, setHistory } from '../redux-actions';
 import { State } from '../redux-reducers';
 
 import './general-page.css';
+import Unauthorized from './Unauthorized';
+import { checkAuthorization } from '../services/ols-service';
 
-const QUERY_ENDPOINT = '/api/proxy/plugin/lightspeed-console-plugin/ols/v1/query';
+const QUERY_ENDPOINT = '/api/proxy/plugin/lightspeed-console-plugin/v1/query';
```

Comment:
I think we need to keep the `ols/` in here to match `proxy.alias`

Reply:
fixed